src/lib/proxy-server.ts

```diff
@@ -81,6 +82,33 @@ export async function startProxyServer(): Promise< boolean > {
 			} );
 		} );
 
+		// On Windows, we need to check if the port is in use before we can listen on it
+		if ( process.platform === 'win32' ) {
+			await new Promise< void >( ( resolve, reject ) => {
+				const tester = createConnection( { port: 80 }, () => {
+					// If we can connect, port is in use
+					tester.end();
+					const error = new Error( 'Port 80 is in use' ) as NodeJS.ErrnoException;
+					error.code = 'EADDRINUSE';
+					reject( error );
+				} );
+
+				tester.setTimeout( 1000, () => {
+					tester.destroy();
+					reject( new Error( 'Port check timed out' ) );
+				} );
+
+				tester.on( 'error', ( err: NodeJS.ErrnoException ) => {
+					if ( err.code === 'ECONNREFUSED' ) {
+						// Port is available
```

Comment:
```suggestion
				tester.on( 'error', ( err ) => {
					if ( isErrnoException( err ) && err.code === 'ECONNREFUSED' ) {
```
Better to adhere to the actual type of `err`